src/io.c

```diff
@@ -85,3 +83,14 @@ chomp(char *string, ssize_t length)
 		string[length - 1] = '\0';
 	}
 }
+
+static char *
+eager_strpbrk(const char *string, const char *separators) {
```

Comment:
That makes a lot of sense. Thanks for explaining it! Should we rename `field_separator` to `field_separators` or `separators` for parity?